drivers/filesystems/CMakeLists.txt

```diff
@@ -2,7 +2,9 @@
 add_subdirectory(btrfs)
 add_subdirectory(cdfs)
 add_subdirectory(ext2)
-#add_subdirectory(fastfat)
+if(SARCH STREQUAL "xbox")
```

Comment:
If this would be enabled for other architectures, could we read xbox disks on those?

Reply:
If the driver is enabled for other architectures, yes, you will be able to read FATX partitions.
However, if you want to read XBOX-partitioned disks (hard-coded partition sizes), you need to use the halxbox.dll HAL.

Reply:
Thanks for the explanation.